### ISSUE TITLE
设置不会保存

### ISSUE THREAD

**Issue by jmlevol** [dipoo/egdownloader#35]
0.95，64位
重开软件要重新设置下载目录、代理等

**Comment by dipoo** [comment]
设置完需要手动点下左侧的【保存】。

**Comment by jmlevol** [comment]
> 设置完需要手动点下左侧的【保存】。

不是。。这点我还是知道的  我下了32位版本和64位两个版本，发现是64位的设置文件setting.xml里只有<?xml version="1.0" encoding="UTF-8"?>这一句，把32位的setting.xml替换过去后就好了
